src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
Added `_isLinkedToArray` but did not add `_arrayAttrRegexps`, which I think only applies to *nested* array attributes like `updatemenus[...].buttons`. Technically this is `transforms[...].styles`, so I'm not totally sure where this would need to be added. At the transform level vs. *inside* styles, perhaps? Glad to add it blindly, but without really knowing how to test it, I think maybe better omit.